lms/services/_helpers/canvas_api.py

```diff
@@ -51,10 +51,31 @@ def token_url(self):
 
         https://canvas.instructure.com/doc/api/file.oauth_endpoints.html#post-login-oauth2-token
 
-        :rtype str:
+        :rtype: str
         """
         return urlunparse(("https", self._canvas_url, "login/oauth2/token", "", "", ""))
 
+    def list_files_url(self, course_id):
+        """
+        Return a Canvas list files API URL for ``course_id``.
+
+        This is the API that returns a list of all the files in a course. See:
+
+        https://canvas.instructure.com/doc/api/files.html#method.files.api_index
+
+        :rtype: str
+        """
+        return urlunparse(
+            (
+                "https",
+                self._canvas_url,
+                f"/api/v1/courses/{course_id}/files",
+                "",
+                urlencode({"content_types[]": "application/pdf", "per_page": 100}),
```

Comment:
So for now we're getting the first 100 files and not supporting pagination. To be addressed later. The default sort order is by name, see https://canvas.instructure.com/doc/api/files.html#method.files.api_index

Reply:
And yes the parameter is really called `content_types[]` with a `[]` on the end